setup/ansible/roles/docker/tasks/main.yml

```diff
@@ -51,7 +51,9 @@
     name: "{{ item }}"
     group: docker
     append: true
-  with_items: "{{ ansible_user }}"
+  with_items:
+   - "{{ ansible_user }}"
+   - pi
```

Comment:
Ubuntu 19.10 doesn't have a user called pi, is this needed? Would it fail if the pi user doesn't exist?

Reply:
Ah, yes, raspbian habits die hard. 
I think I saw `vars.example.yml` had `ansible_user: pi` and hardcoded it a it.

Reply:
I will update vars.example.yml with a user `ubuntu` instead of `pi`, but `ubuntu` user should be handled as well in the `ansible_user` variable. We only need the one user in this list (or revert to no list)